test/cppad/cg/CppADCGOperationTestImpl.hpp

```diff
@@ -82,7 +82,12 @@ void CppADCGOperationTest::compile(const std::string& source, const std::string&
          *   -Wl,-soname, library  Pass suitable options to linker
          *
          */
-        std::string linker = "-Wl,-soname," + library;
```

Comment:
We can improve the entire function (make it much smaller).
See:
https://github.com/joaoleal/CppADCodeGen/blob/feature/parameters/test/cppad/cg/CppADCGOperationTestImpl.hpp